tensorflow/workspace.bzl

```diff
@@ -573,11 +573,10 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   patched_http_archive(
       name = "boringssl",
       urls = [
-          "http://mirror.bazel.build/github.com/google/boringssl/archive/bbcaa15b0647816b9a1a9b9e0d209cd6712f0105.tar.gz",
-          "https://github.com/google/boringssl/archive/bbcaa15b0647816b9a1a9b9e0d209cd6712f0105.tar.gz",  # 2016-07-11
```

Comment:
Please also add the mirror.bazel.build link.
We take care of mirroring the URLs.
@jart could you help with the mirror here?

Reply:
Done.